marklogic-client-api/src/test/java/com/marklogic/client/test/RowManagerTest.java

```diff
@@ -523,7 +523,7 @@ public void testSQLException() {
     RowManager rowMgr = Common.client.newRowManager();
     PlanBuilder p = rowMgr.newPlanBuilder();
     PlanBuilder.ExportablePlan builtPlan =
-        p.fromSql("select case when lastName = 'Davis' then fn_error(fn_qname('', 'SQL-TABLENOTFOUND'), 'Internal Server Error') end, opticUnitTest.musician.* from (select * from opticUnitTest.musician order by lastName)");
```

Comment:
I just checked the Jenkins job, this test passes in 11, but failed in 10. It should be excluded from running in ML 10. We should add an annotation as you mentioned, but not modify it I think.

Reply:
Thanks @llinggit - given that it's 1 of 3 tests passing on 11 but not on 10, I'm going to remove this change completely and tackle all 3 tests in a separate JIRA / PR.